reccmp/isledecomp/parser/codebase.py

```diff
@@ -13,13 +14,13 @@
 
 
 class DecompCodebase:
-    def __init__(self, filenames: Iterable[str], module: str) -> None:
+    def __init__(self, paths: Iterable[Path], module: str) -> None:
         self._symbols: list[ParserSymbol] = []
 
         parser = DecompParser()
-        for filename in filenames:
-            parser.reset_and_set_filename(filename)
-            with open(filename, "r", encoding="utf-8") as f:
+        for path in paths:
+            parser.reset_and_set_filename(str(path))
```

Comment:
This looks like `parser.reset_and_set_filename()` should take an argument of type `Path`, or am I missing something?